config/ant-phase-verify.xml

```diff
@@ -60,7 +60,7 @@
     <tstamp>
       <format property="FINISHED" pattern="dd/MM/yyyy hh:mm:ss aa"/>
     </tstamp>
-    <echo level="info">Checkstyle finished (checkstyle-checks.xml) : ${FINISHED}</echo>
```

Comment:
main:


fix:
<img width="1831" alt="image" src="https://github.com/user-attachments/assets/34ce9a15-cbe5-4585-8be8-9cf9111c563c" />

Reply:
one more location needs removal of this space
https://github.com/search?q=repo%3Acheckstyle%2Fcheckstyle%20%22Checkstyle%20finished%22&type=code